proof_of_concept/ddm_site.py

```diff
@@ -31,18 +31,19 @@ def __init__(
 
         Args:
             name: Name of the site
-            administrator: Party which administrates this site.
+            owner: Party which owns this site.
             namespace: Namespace used by this site.
             stored_data: Data sets stored at this site.
             rules: A policy to adhere to.
 
         """
         # Metadata
         self.name = name
-        self.administrator = administrator
+        self.owner = owner
```

Comment:
If owner == administrator, what's the point of having a distinction?

Reply:
The owner owns the namespace for this site (i.e. has the right to determine what to do with the assets in the namespace), while the administrator does the technical system administration of the site, and is therefore the party you need to trust in order to give access to your assets.

For now owners and administrators are the same party, but the idea here is that there may be providers offering a DDM-as-a-Service, in which case their customers would be owners, but the provider would be the administrator, so I'd like to keep them separate here so as to make this a more realistic POC.

Reply:
Cool! Maybe add a comment with this info?

Reply:
Yes, good idea.